packages/eas-cli/src/update/android/UpdatesModule.ts

```diff
@@ -15,9 +18,11 @@ export async function syncUpdatesConfigurationAsync(
   ensureValidVersions(exp, RequestedPlatform.Android);
   const accountName = (await getOwnerAccountForProjectIdAsync(graphqlClient, projectId)).name;
 
+  AndroidConfig.Updates.withUpdates(exp, { expoUsername: null });
```

Comment:
what is this line doing?

I might not understand how plugin stuff works, but when see what is inside withUpdates it leads me to think that it's either is doing the same stuff as the code bellow, so it does the same thing twice, or it only evaluates config with plugin which would mean it does not do anything because plugins were already evaluated in exp that is passed to the function.

Reply:
Oh, sorry, this was just me testing. I actually have no idea how to call a config plugin.